### ISSUE TITLE
Key file names

### ISSUE THREAD

**Issue by namgang** [lifeisfoo/emacs-grails#15]
I'm using 0.4.0 with Grails 3.1.5. Two file names seem to be incorrect.
- grails-urlmappings-by-version should be "controllers/_appName_/UrlMappings.groovy" where _appName_ is the name of the Grails app
- grails-bootstrap-by-version should be "init/BootStrap.groovy" (capital "S")

**Comment by lifeisfoo** [comment]
You're right, there are two bugs here:
- wrong camelCase names (I guess that this is working for me since now I'm using hfs that is _case insensitive but case preserving_: a really strange osx feature, but if you want you can call it _bug_);
- UrlMappings location ignore the application namespace (only in grails 3+)

**Comment by lifeisfoo** [comment]
The first is fixed in 0.4.1. 
For the last one I've created a separated issue since it requires more work #16.
Please subscribe to the new issue to keep you updated.
I hope this mode is helping you. Thank you for reporting these bugs.

**Comment by namgang** [comment]
Great, this is a nifty little thing for people who live and thrive in Emacs. Like I do...

```
On Sunday, April 17, 2016 8:46 AM, Alessandro Miliucci <notifications@github.com> wrote:
```

 The first is fixed in 0.4.1. 
For the last one I've created a separated issue since it requires more work #16.
Please subscribe to the new issue issue to keep you updated.
I hope this mode is helping you. Thank you for reporting these bugs.—
